codespell_lib/tests/test_dictionary.py

```diff
@@ -86,30 +83,40 @@ def _check_aspell(phrase, msg, in_aspell, fname, languages):
         assert not this_in_aspell, '%s should not %s' % (msg, end)
 
 
+whitespace = re.compile(r'\s')
+start_whitespace = re.compile(r'^\s')
+start_comma = re.compile(r'^,')
+whitespace_comma = re.compile(r'\s,')
+comma_whitespaces = re.compile(r',\s\s')
+comma_without_whitespace = re.compile(r',[^ ]')
```

Comment:
Given this isn't all whitespace, just an actual space, can I suggest this change (and the equivalent below):
```suggestion
comma_without_space = re.compile(r',[^ ]')
```